src/main/java/org/traccar/protocol/Gt06ProtocolDecoder.java

```diff
@@ -794,22 +794,34 @@ private Object decodeBasic(Channel channel, SocketAddress remoteAddress, ByteBuf
 
             position.set(Position.KEY_TYPE, type);
 
+            if (type == MSG_GPS_LBS_2 && "NT20".equals(model)) {
```

Comment:
Let's maybe create a variable for `"NT20".equals(model)`, so we don't need to evaluate it so many times. Like `isNt20` or something.